### DISCUSSION TITLE
help: can i catch keyboard shortcuts in slint ui

### DISCUSSION THREAD

**Discussion by heng30** [slint-ui/slint#2580] [Q&A]

**Comment by ghost** [comment]
Hi, 
yes it's possible. You can read the keyboard modifiers of a KeyEvent, that e.g. is used as parameter of the callbacks of FocusScope https://slint-ui.com/releases/1.0.0/docs/slint/src/builtins/structs.html#keyevent.

**Reply by nic-obert** [reply]
The updated documentation link: https://docs.slint.dev/latest/docs/slint/reference/keyboard-input/focusscope/

**Comment by heng30** [comment]
Thank you for your reply.
